electrum/gui/qt/theme_helper.py

```diff
@@ -0,0 +1,49 @@
+from PyQt5.QtWidgets import QWidget, QApplication
+
+from electrum.gui.qt.stylesheet_patcher import patch_qt_stylesheet
+from electrum.gui.qt.util import  ColorScheme
+
+def set_qtum_theme_if_needed(self):
```

Comment:
better rename `self` to `gui` or `electrum_gui`

Reply:
Ok

Reply:
Did another emthod,just passed config to the theme_helper instead.Marking as resolved.